btree/btree.c

```diff
@@ -11636,6 +11648,8 @@ static int ut_btree_suite_init(void)
 	m0_be_ut_seg_init(ut_seg, ut_be, BE_UT_SEG_SIZE);
 	seg = ut_seg->bus_seg;
 
+	g_process_fid = g_process_fid;
```

Comment:
Is this added to avoid compilation error?

Reply:
yes 👍